examples/playground/json-examples/3d-heatmap.json

```diff
@@ -1,6 +1,7 @@
 {
   "description": "The deck.gl website hexagonlayer example in JSON format",
   "websiteUrl": "https://deck.gl/#/examples/core-layers/hexagon-layer",
+  "controller": "MapController",
```

Comment:
Not sure it makes sense to add this to this example?

If we do, we should also edit `playground/configuration.js`?